index.html

```diff
@@ -27,7 +27,7 @@
 		<div class="container">
 			<br/>
 			<h1><strong> Rahul Zhade </strong></h1>
-			<h3> AppSec Intern at <strong> Github </strong> </h3>
+			<h3> AppSec Engineer at <strong> Github </strong> </h3>

```

Comment:
I can't believe they hired me with this typo on here 👀 

```suggestion
			<h3> AppSec Engineer at <strong> GitHub </strong> </h3>

```